core/domain/exp_jobs_one_off_test.py

```diff
@@ -1231,6 +1232,41 @@ def test_for_validation_job(self):
         self.assertEqual(actual_output, expected_output)
 
 
+class DeleteImagesFromGAEJobTest(test_utils.GenericTestBase):
+
+    COMMITER_ID = 'ADMIN'
+    COMMIT_MESSAGE = 'Deleting file_model for image from GAE'
+    EXP_ID = 'eid'
+    FILENAME = 'imageFile.png'
+
+    def setUp(self):
+        super(DeleteImagesFromGAEJobTest, self).setUp()
+        self.process_and_flush_pending_tasks()
+
+    def test_for_deletion_job(self):
+        """Checks that images get deleted from the GAE after running the job.
+        """
+        fs = fs_domain.AbstractFileSystem(
+            fs_domain.ExplorationFileSystem(self.EXP_ID))
+        imageData = ''
+        mimetype = 'image/png'
+        fs.commit(
+            self.COMMITER_ID, self.FILENAME, imageData,
+            mimetype=mimetype)
+
+        job_id = exp_jobs_one_off.DeleteImagesFromGAEJob.create_new()
```

Comment:
Before and after running the job, you should also try retrieving the image in question.

Reply:
Done!